extensions/ql-vscode/src/vscode-tests/minimal-workspace/databases.test.ts

```diff
@@ -391,145 +378,146 @@ describe("databases", () => {
     it("should handle an empty file", () => {
       const db = createMockDB(Uri.parse("file:/sourceArchive-uri/"));
       const resolved = db.resolveSourceFile("");
-      expect(resolved.toString()).to.eq("file:///sourceArchive-uri/");
+      expect(resolved.toString()).toBe("file:///sourceArchive-uri/");
     });
   });
 
   it("should not support the primary language", async () => {
-    supportsLanguageNameSpy.resolves(false);
+    supportsLanguageNameSpy.mockResolvedValue(false);
 
     const result = await (databaseManager as any).getPrimaryLanguage("hucairz");
-    expect(result).to.be.undefined;
+    expect(result).toBeUndefined();
   });
 
   it("should get the primary language", async () => {
-    supportsLanguageNameSpy.resolves(true);
-    resolveDatabaseSpy.resolves({
+    supportsLanguageNameSpy.mockResolvedValue(true);
+    resolveDatabaseSpy.mockResolvedValue({
       languages: ["python"],
-    });
+    } as unknown as DbInfo);
     const result = await (databaseManager as any).getPrimaryLanguage("hucairz");
-    expect(result).to.eq("python");
+    expect(result).toBe("python");
   });
 
   it("should handle missing the primary language", async () => {
-    supportsLanguageNameSpy.resolves(true);
-    resolveDatabaseSpy.resolves({
+    supportsLanguageNameSpy.mockResolvedValue(true);
+    resolveDatabaseSpy.mockResolvedValue({
       languages: [],
-    });
+    } as unknown as DbInfo);
     const result = await (databaseManager as any).getPrimaryLanguage("hucairz");
-    expect(result).to.eq("");
+    expect(result).toBe("");
   });
 
   describe("isAffectedByTest", () => {
     const directoryStats = new fs.Stats();
     const fileStats = new fs.Stats();
-
-    before(() => {
-      sinon.stub(directoryStats, "isDirectory").returns(true);
-      sinon.stub(fileStats, "isDirectory").returns(false);
-    });
+    jest.spyOn(directoryStats, "isDirectory").mockReturnValue(true);
+    jest.spyOn(fileStats, "isDirectory").mockReturnValue(false);
```

Comment:
Does this not need to be called in `beforeEach` since mocks are reset before every test?